aristamd/src/app/components-section/accordion-demo/accordion-demo.component.js

```diff
@@ -0,0 +1,50 @@
+class AccordionDemoController {
+  constructor($scope, $uibModal, $timeout, communicationCenterService) {
+    const ctrl = this;
+
+    ctrl.$onInit = function () {
+
+      ctrl.headerContent1 = '<accordion-header-1></accordion-header-1>';
+      ctrl.headerContent2 = '<accordion-header-2></accordion-header-2>';
+      ctrl.bodyContent = require('./accordion-content.template.html');
+
+       ctrl.options = [
+      {
+        name:'title',
+        type:'string',
+        description:''
+      },
+      {
+        name:'steps',
+        type:'array',
+        description:''
+      },
+      {
+        name:'stepperApi',
+        type:'object',
+        description:''
+      },
+      {
+        name:'id',
+        type:'string',
+        description:''
+      },
+      ]
+    }
+
+    ctrl.open = function (size, parentSelector) {
+      var modalInstance = $uibModal.open({
+        animation: true,
+        component: 'stepperModal',
+        size: size,
```

Comment:
No need to specify the variable when both property and variable has the same name.